newIDE/app/src/UI/EditorMosaic/index.js

```diff
@@ -223,13 +236,18 @@ export default class EditorMosaic extends React.Component<Props, State> {
               }
 
               return (
-                <MosaicWindow
-                  path={path}
-                  title={editor.title}
-                  toolbarControls={editor.toolbarControls}
-                >
-                  {editor.renderEditor()}
-                </MosaicWindow>
+                <I18n>
+                  {({ i18n }) => (
+                    <MosaicWindow
+                      path={path}
+                      title={editor.title}
+                      tooltip={i18n._(editor.tooltip)}
```

Comment:
What if `tooltip` is not defined? You should either disallow this or handle this case.

Reply:
`tooltip={editor.tooltip ? i18n._(editor.tooltip) : null}` ?

With tooltip
![WhatsApp Image 2020-03-06 at 2 35 17 AM](https://user-images.githubusercontent.com/39851078/76025938-e4933b80-5f53-11ea-8ab0-d97dd42b5bd7.jpeg)

Without tooltip
![WhatsApp Image 2020-03-06 at 2 35 20 AM](https://user-images.githubusercontent.com/39851078/76025975-f4ab1b00-5f53-11ea-9e6e-5fd83c043d70.jpeg)